src/pseudopeople/default_configuration.yaml

```diff
@@ -0,0 +1,200 @@
+# Default noising configuration
```

Comment:
This is very, very long. Do we want a default config per form?

Reply:
RT discussion needed.